homeassistant/components/modbus/base_platform.py

```diff
@@ -165,7 +166,9 @@ def __init__(self, hub: ModbusHub, config: dict) -> None:
         if self._scale < 1 and not self._precision:
             self._precision = 2
         self._offset = config[CONF_OFFSET]
-        self._slave_count = config.get(CONF_SLAVE_COUNT, 0)
+        self._slave_count = config.get(
+            CONF_SLAVE_COUNT, config.get(CONF_VIRTUAL_COUNT, 0)
+        )
```

Comment:
Could use the `or` operator here too.

Reply:
Of course, done, good that you are awake 😄